dataplicity/m2m/bencode.py

```diff
@@ -60,12 +61,33 @@ def encode(obj):
 
     def add_encode(obj):
         if isinstance(obj, bytes):
-            append(b"{}:{}".format(len(obj), obj))
+            if PY2:
+                append(b"{}:{}".format(len(obj), obj))
+            elif PY3:
+                append(six.b("{}:".format(len(obj))))
+                append(obj)
         elif isinstance(obj, text_type):
             obj = obj.encode('utf-8')
-            append(b"{}:{}".format(len(obj), obj))
+            if PY2:
```

Comment:
This should work for both Py versions. Best to avoid that additional function call.

```
append(u"{}:{}".format(len(obj), obj).encode())
```